cloudera-integration/csd/scripts/control.sh

```diff
@@ -39,6 +39,9 @@ export DEFAULT_LOGBACK_CONFIG="${CONF_DIR}/logback.xml" # This is auto-generated
 export LOGBACK_CONFIG=${LOGBACK_CONFIG:-$DEFAULT_LOGBACK_CONFIG}
 export KEYTAB_FILE=${KEYTAB_FILE:-"${CONF_DIR}/pulse.keytab"}
 export AKKA_CONF=${AKKA_CONF:-"application.conf"}
+export ALERT_ENGINE_MAX_HEAP=${ALERT_ENGINE_MAX_HEAP:-"2G"}
```

Comment:
Should this also be in bytes?

Reply:
This is fine, java can read this. But the default if you don't add a `G` or an `M` is in bytes